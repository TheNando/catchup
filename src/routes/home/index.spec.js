import { h } from 'preact'
import { shallow } from 'preact-render-spy'
import Home from './index'

describe('Home', () => {
  it('renders the component', () => {
    const context = shallow(<Home />)
    expect(context.output()).toMatchSnapshot()
  })
})
