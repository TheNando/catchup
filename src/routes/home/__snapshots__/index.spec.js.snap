// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the component 1`] = `
<div class="page">
  <Timer start="25:00"></Timer>
</div>
`;
